.circleci/src/@common.yml

```diff
@@ -5,6 +5,10 @@ parameters:
     default: true
     type: boolean
 
+  run_commit_main:
+    default: false
+    type: boolean
+
```

Comment:
Why is this needed?

Reply:
it was just for testing. sorry didn't draft the pr